### DISCUSSION TITLE
Limits page says limits are applied to both cloud & self-hosted versions, is this correct?

### DISCUSSION THREAD

**Discussion by Warchant** [triggerdotdev/trigger.dev#694] [Q&A] [answered]
https://trigger.dev/docs/documentation/concepts/limits
>The following limits apply to the Trigger.dev Cloud service and users of the self-hosted version of Trigger.dev.

Is this sentence correct? If so, why self-hosted version has limits?

**Comment by matt-aitken** [comment, marked as answer]
The self hosted version is effectively the same as the Enterprise version. So, when self-hosting there are no limits other than the technical limits of wherever you deploy to.

**Reply by Warchant** [reply]
Thank you for the answer.
I clearly misunderstood that page and missed that column header: 
![image](https://github.com/triggerdotdev/trigger.dev/assets/1867551/6ae3b20d-e007-483f-92be-c9c44dd15e5f)

P.S. awesome product!
